src/controllers/emailVerifier.js

```diff
@@ -0,0 +1,84 @@
+import models from '../db/models';
+
+const { User, VerificationToken } = models;
```

Comment:
@waleCloud you're right that it can be managed within the user's table. I really didn't think of this your implementation. I was looking at the verification token as a model which it truly isn't.
I will refactor.